tests/testPhase.cpp

```diff
@@ -58,7 +58,7 @@ TEST(Phase, All) {
 
   // contactPointObjectives
   const Point3 step(0, 0.4, 0);
-  const gtsam::SharedNoiseModel cost_model = nullptr;
+  const gtsam::SharedNoiseModel cost_model = gtsam::noiseModel::Unit::Create(3);
```

Comment:
why?

Reply:
ExpressionFactors won't allow noiseModel to be nullptr